sqlcontacts.py

```diff
@@ -1,12 +1,60 @@
+'''
+This a console-based application
+'''
+import sqlite3
+
+
 class PhoneBook:
+    '''
+    Class for working only with phone book db
+    Have methods:
+        save(), create(name, phone), read(name),
+        update(name, phone), delete(name)
+    '''
+    def __init__(self):
+        self.conn = sqlite3.connect(":memory:")
+        self.cursor = self.conn.cursor()
+        self.cursor.execute("""CREATE TABLE phone_data
+                          (Name_contact text, Number_contact text)""")
+
     def create(self, name, phone):
-        pass
+        '''Create new contact and save in db'''
+        sql = "SELECT * FROM phone_data WHERE Name_contact=?"
+        self.cursor.execute(sql, (name,))
+        if self.cursor.fetchall() == []:
+            sql_insert = "INSERT INTO phone_data VALUES (?, ?)"
+            self.cursor.execute(sql_insert, (name, phone))
+            return "create contact"
+        raise KeyError
 
     def read(self, name):
-        pass
+        '''Read contact from db'''
+        sql = "SELECT * FROM phone_data WHERE Name_contact=?"
+        self.cursor.execute(sql, (name,))
+        if self.cursor.fetchall() != []:
```

Comment:
if self.cursor.fetchall():
...
Also it's better to use fetchone() in this case